jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java

```diff
@@ -8,7 +9,7 @@
 import com.epam.jdi.light.ui.html.elements.common.Icon;
 import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
 import io.github.com.pages.sections.CheckboxSection;
-import io.github.com.pages.sections.InputSection;
```

Comment:
What is the reason for removing the InputSection?

Reply:
Fixed, thanks.